content/Video_Display.md

```diff
@@ -738,7 +738,7 @@ Mode 0 - H-Blank Period
 Mode 1 - V-Blank Period
 ```
 
-Aside from that, OAM can be accessed at any time by using the DMA
+During those modes, OAM can be accessed at any time by using the DMA
```

Comment:
I'd further clarify that OAM DMA out-prioritizes the PPU in accessing OAM, and that the PPU will read `$FF` from OAM during that time. I'd thus make it a separate paragraph, to make it stand out.